projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/read-query-management-field.component.ts

```diff
@@ -34,4 +35,9 @@ export class ReadQueryManagementFieldComponent extends AbstractFieldReadComponen
       QueryManagementUtils.extractCaseQueriesFromCaseField();
     }
   }
+
+  public setQuery(query) {
```

Comment:
@DavidJayakumar - please add return type for this function

Reply:
as per Dan's comment

Reply:
done